finality-aleph/src/sync/mock/backend.rs

```diff
@@ -202,6 +193,19 @@ impl Finalizer<MockJustification> for Backend {
         };
 
         block.finalize(justification);
+
+        // Check if the previous block was finalized, or this is the last block of the current
+        // session
+        let allowed_numbers = match storage.finalized.last() {
+            Some(id) => [
+                id.number + 1,
+                id.number + storage.session_period
+                    - (id.number + 1).rem_euclid(storage.session_period),
+            ],
+            None => [0, storage.session_period - 1],
+        };
+        assert!(allowed_numbers.contains(&id.number));
```

Comment:
Same comment as above, make this print something reasonable on failure plx.

Reply:
fixed

Reply:
Same here.

Reply:
same here